src/navigators/createStackNavigator.tsx

```diff
@@ -2,8 +2,24 @@ import { StackRouter, createNavigator } from '@react-navigation/core';
 import { createKeyboardAwareNavigator } from '@react-navigation/native';
 import { Platform } from 'react-native';
 import StackView from '../views/StackView/StackView';
+import { NavigationStackOptions, NavigationProp, Screen } from '../types';
 
-function createStackNavigator(routeConfigMap, stackConfig = {}) {
+function createStackNavigator(
+  routeConfigMap: {
+    [key: string]:
+      | Screen
+      | {
+          screen: Screen;
```

Comment:
another option here, which is an alternative to `screen`, is `getScreen`. it is a function that returns the screen component, eg `getScreen: () => { return require('./screens/Thing').default }`